src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -17,7 +17,7 @@ import { QueryStateModel } from '../../models/QueryStateModel';
 import { IAttributesChangedEventArg, MODEL_EVENTS } from '../../models/Model';
```

Comment:
I would pay too much attention to this file, there will be a lot of refactor here once I remove the old value management

Reply:
It doesn't make 100% of sense for now